ckanext/restricted/templates/restricted/restricted_request_access_base.html

```diff
@@ -1,3 +1,4 @@
+{% set pkg_dict = pkg_dict or {} %}
```

Comment:
Instead a better way would be to pass empty dict as `pkg_dict` from the controller:
```
def _send_organization_request(self, context):
[....]
        return render(
            'restricted/restricted_request_organization_result.html',
            extra_vars={'data': data_dict, 'pkg_dict': {}, 'success': success}
        )
```

Reply:
Sure, I tried to do a capture-all solution as I wasn't sure why restricted even uses the `package/base.html` for creating an organization. So I kinda assumed it might be better to patch it at the top of the wrongly imported template 😅

Reply:
I've tested it this way and it seems to work, so I guess it's ok